typings/index.d.ts

```diff
@@ -2195,15 +2171,14 @@ declare module 'discord.js' {
     RESOURCE_OVERLOADED: 130000;
   }
 
-  type APIMessageContentResolvable = string | number | boolean | bigint | symbol | readonly StringResolvable[];
-
   interface ApplicationAsset {
     name: string;
     id: Snowflake;
     type: 'BIG' | 'SMALL';
   }
 
-  interface AuditLogChange {
+    type APIMessageContentResolvable = string | number | boolean | bigint | symbol | readonly string[];
```

Comment:
Is this even needed? I thought the point was to drop all these, and stick to string?

Reply:
should be solved in https://github.com/discordjs/discord.js/pull/4880/commits/c20bf396e987b2f8a8c8eb9cdec471f5ed245ca8 - moved the tests (like primitive symbol and non-options object to expect-error, since these introduced tests are precisely not what's wanted after this PR; removed the newly introduced type APIMessageContentResolvable, since it was just used in edit, where now a string is expected anyways)